Formula/zimpl.rb

```diff
@@ -0,0 +1,25 @@
+class Zimpl < Formula
+  desc "Language to translate the mathematical model of a problem into a MI(N)LP"
+  homepage "https://zimpl.zib.de"
+  url "https://scipopt.org/download/release/scipoptsuite-8.0.3.tgz"
+  version "3.5.3"
```

Comment:
Is there no direct download?

Reply:
Unfortunately, not. Only old versions are downloadable through https://zimpl.zib.de.

Reply:
```
zimpl: livecheck error
Unable to get versions
```